pkg/operation/seed/seed.go

```diff
@@ -268,6 +279,10 @@ func BootstrapCluster(seed *Seed, secrets map[string]*corev1.Secret, imageVector
 		credentials := deployedSecretsMap["seed-logging-ingress-credentials"]
 		basicAuth = utils.CreateSHA1Secret(credentials.Data[utilsecrets.DataKeyUserName], credentials.Data[utilsecrets.DataKeyPassword])
 		kibanaHost = seed.GetIngressFQDN("k", "", "garden")
+
+		sgFluentdCredentials := deployedSecretsMap["fluentd-es-sg-credentials"]
+		sgFluentdPassword = string(sgFluentdCredentials.Data["password"])
+		sgFluentdPasswordHash = string(sgFluentdCredentials.Data["hashedPassword"])
```

Comment:
```suggestion
		sgFluentdPasswordHash = string(sgFluentdCredentials.Data["bcryptPasswordHash"])
```

or something similar